modules/tensor_mechanics/src/materials/ComputeMultipleInelasticStress.C

```diff
@@ -75,6 +75,10 @@ ComputeMultipleInelasticStress::ComputeMultipleInelasticStress(const InputParame
     _elasticity_tensor(getMaterialPropertyByName<RankFourTensor>(_base_name + "elasticity_tensor")),
     _elastic_strain_old(getMaterialPropertyOld<RankTwoTensor>(_base_name + "elastic_strain")),
     _strain_increment(getMaterialProperty<RankTwoTensor>(_base_name + "strain_increment")),
+    _has_compliance(hasMaterialProperty<RankFourTensor>(_base_name + "compliance_tensor")),
```

Comment:
Uhm, you are doing `hasMaterialProperty` in the constructor again. This introduces a construction order dependence which we really want to avoid!